src/DependencyInjection/Configuration.php

```diff
@@ -145,6 +147,9 @@ function (string $value): string {
                         ->arrayNode('imagine_options')
                             ->addDefaultsIfNotSet()
                             ->children()
+                                ->scalarNode('class')
```

Comment:
The `imagine_options` array gets directly passed to `setImagineOptions()` https://github.com/contao/image/blob/dc9e2dc78533c6da1d7d0cf010b0b89b2b5ffd06/src/ResizeOptionsInterface.php#L29 so we should probably not add the class name here.

IMO it would be better to use `contao.image.imagine_class` instead.

Reply:
Agrred